pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -33,6 +34,10 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
+// Changes outside of the kubernetes universe could cause the approval decision
+// to change. We get around this be frequently resyncing.
```

Comment:
Picked 15 seconds for initial testing but I was hoping to discuss what would be reasonable in this review. What do you think?